### PR TITLE
Embed PDBs in built binary

### PR DESCRIPTION
**Pull request by KevinRansom**
This PR embeds compressed PDBs into the built binary.
It adds embedded as an option to --debug
updates command line tests.

### CODE REVIEW

**Comment by dsyme** [comment]
"This branch has conflicts that must be resolved"?

File: src/absil/ilwritepdb.fs [hunk 1]
```diff
@@ -86,44 +87,70 @@ type PdbData =
       Methods: PdbMethodData[] 
       TableRowCounts: int[] }
 
-//---------------------------------------------------------------------
-// Portable PDB Writer
-//---------------------------------------------------------------------
+type BinaryChunk =
```
Comment (KevinRansom) [inline src/absil/ilwritepdb.fs]: Put this in ilwrite.fsi.  It is not pdb specific.

Comment (KevinRansom) [inline src/absil/ilwritepdb.fs]: foo

File: tests/fsharpqa/Source/CompilerOptions/fsc/help/help40.437.1033.bsl [hunk 2]
```diff
@@ -1,110 +1,112 @@
...
---linkresource:<resinfo>       Link the specified resource to this assembly
-                               where the resinfo format is <file>[,<string
-                               name>[,public|private]]
+--win32res:<file>                        Specify a Win32 resource file (.res)
+--win32manifest:<file>                   Specify a Win32 manifest file
+--nowin32manifest                        Do not include the default Win32 manifest
+--resource:<resinfo>                     Embed the specified managed resource
+--linkresource:<resinfo>                 Link the specified resource to this assembly
+                                         where the resinfo format is <file>[,<string
+                                         name>[,public|private]]
```
Comment (dsyme) [inline tests/fsharpqa/Source/CompilerOptions/fsc/help/help40.437.1033.bsl]: Does the help text still look OK visually when printed on an 80-column console window?  thanks

**Comment by dsyme** [comment]
@KevinRansom Please add at least one invocation of the compiler that uses ``--debug:embedded`` to the automated tests? Also jot down the manual sanity checks that have been done? thanks

**Comment by KevinRansom** [comment]
1:  I will add a smoke test when I have a machine that build again ... I am paying a steep price for having to install the latest internal VS builds right now.
2:  Verification Built and debugged F# compiler (Net40 debug build): 
. Full pdbs
. Portable pdbs
and Embedded pdb


 full pdbs.

**Pull request merged**